### DISCUSSION TITLE
Definición de structs

### DISCUSSION THREAD

**Discussion by cebabeltran** [IIC2133-PUC/2024-1#10] [Tarea 0] [answered]
Holaaa, tengo una duda con respecto a la creación de structs en la tarea. Las partes que dicen "completar" la implementación y construcción de estructuras están integradas dentro del flujo de main{}, pero en las ayudantías de C vimos como crear structs fuera del flujo (con typedef struct nombre {}) ¿Hay alguna diferencia entre hacerlo de un modo u otro? ¿Y cuál sería el recomendado? Gracias!

**Comment by PabloNavarroY** [comment, marked as answer]
Las estructuras se **definen** fuera del flujo ocupando, como bien dices, `typedef` y `struct`. Con construcción de la estructura nos referimos a lo que en OOP se llama instanciación (aunque C no está orientado a objetos) y eso se suele hacer dentro del flujo.
